pulp_smart_proxy/app/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.urls import path
+
+from pulpcore.app.views import StatusView  # the plugin wants to expose status
+
+from .views import FeaturesView, FeaturesV2View, VersionView
+
+if settings.DOMAIN_ENABLED:
+    V3_API_ROOT = settings.V3_DOMAIN_API_ROOT_NO_FRONT_SLASH
+else:
+    V3_API_ROOT = settings.V3_API_ROOT_NO_FRONT_SLASH
+
+urlpatterns = [
+    path(f"{V3_API_ROOT}smart_proxy/features", FeaturesView.as_view()),
+    path(f"{V3_API_ROOT}smart_proxy/v2/features", FeaturesV2View.as_view()),
+    path(f"{V3_API_ROOT}smart_proxy/version", VersionView.as_view()),
+    path(f"{V3_API_ROOT}smart_proxy/pulpcore/status", StatusView.as_view()),
```

Comment:
I have no idea why our current proxy plugin exposes/proxies `/status`, but let's copy that for now?

Reply:
Added in https://github.com/theforeman/smart_proxy_pulp/commit/cdf3e7191fe5745cbc470634fa650c3ceed17f69 without a lot of explanation, but you can see that it was also done in Pulp 2. It's a good question whether it's actually used.

Reply:
Looking at Katello itself I don't think it actually called: https://github.com/Katello/katello/blob/6db538659a93349c0bc63419d78769416ed963a7/app/models/katello/ping.rb#L254-L265 uses `#{url}/status` and then we see it called on the `pulp3_url` itself. https://github.com/Katello/katello/blob/6db538659a93349c0bc63419d78769416ed963a7/app/models/katello/ping.rb#L201-L209 is called from https://github.com/Katello/katello/blob/6db538659a93349c0bc63419d78769416ed963a7/app/models/katello/ping.rb#L66-L70

So we can see `url` resolves to `fetch_proxy(capsule_id).pulp3_url`. Interestingly enough it also calls it for content on the same URL.

Reply:
It does! (Guess who implemented it ;)
But it does so, because it asks the Pulp API whether the Content App is up.